draft-rfcs/working-group-terminology.md

```diff
@@ -2,96 +2,274 @@
+It is recommended for the working group to create a `wg-{name}`/`project-{name}`
+repository under the `rust-lang` organisation using the [project group
+template], and making any relevant changes and personalisation.
 
-- Creation of a Working Group is approved by the Core team.
+[`rust-lang/team`]: https://github.com/rust-lang/team
+[project group template]: https://github.com/rust-lang/project-group-template
 
-### Project Groups
+## Working Groups
 
-A Project Group is a group of people working on a particular project or responsibilities at the behest of an official Rust team. Examples of this would include FFI Unwind, Triage, or The Rust Survey.
+Working groups are a way of finding new domains that would benefit from
+continual development, that are not covered by an existing Rust team. Examples
+of this include [Embedded][embedded-wg], [WebAssembly][wasm-wg], and
+[Game Development][gamedev-wg].
 
-- Creation of a Project Group is approved by the related team.
+[embedded-wg]: https://github.com/rust-embedded/wg
+[wasm-wg]: https://github.com/rustwasm/team
+[gamedev-wg]: https://github.com/rust-gamedev/wg
 
-### Community Groups
+### Working Group Creation
 
-Community Groups are groups of individuals who want to work and collaborate on a specific topic, without going through the formal process of a working group. Community groups are free to adopt the structure of working or projects groups or create their own structure.
+Creation of a working group is approved by the core team. Typically this has
+been done by the core team agreeing to approve the creation of new working
+groups and having a period of time soliciting applications from the community,
+and then approving a subset of those applications.
 
-- Community groups are not required to follow any of the guidelines mentioned above.
-- As such community groups are not officially endorsed by the Rust project.
+> **DRAFT NOTE** Should this application come in the form of an RFC? My
+> inclination is yes, however it could just create more churn and drama than
+> needed. Posting in a thread on internals as was done previously might
+> be enough.
 
-# Drawbacks
+#### Application Checklist
+
+This not meant to be formal list of questions to be answer, however the
+application should try to address these points.
+
+- What value do you see your group bringing to the organisation?
+- Why should the project your group?
+- What are the goals of your group?
+- What are explicitly non-goals of your group?
+
+These first four points should also serve as the basis for the charter for the
+working group.
 
-[drawbacks]: #drawbacks
+- Is your group long-running or temporary?
+  - If it is temporary, how long do you see it running for?
+- What is the long-term vision of your group?
+- What do you expect the relationship to the organisation be?
```

Comment:
Folks can have a hard time coming to a shared understanding of terms like vision and mission, and it feels like there is a big gap between "short-term goals" and "long-term vision". Describing group activities in a more explicitly-structured way could be helpful.  It might also be useful to put the current contents of lines 161 and 166 next to each other to help folks integrate the different levels of their statement of purpose. Also, examples may be helpful. New version of lines 161 and 166 with an additional middle level below:

What vision does your group want to realize?
Example: Official Rust groups are organized in a sustainable, clearly described way.

What are some of the core objectives that support that vision? 
Example: Creating a clear framework for group types and ensuring that group-related infrastructure is up to date. 
What are your group's short-term goals (for its first month or first year, depending on group scope)? Creating an RFC describing group types and the differences between them; Auditing existing repositories for recent activity.

These examples aren't perfect, but hopefully they provide some guidance. They can of course be tweaked or replaced.